english/get-away/get-away.md

```diff
@@ -1,2 +1,8 @@
 You race across the rickety bridge as the officer explodes from the room.  He begins
-across the bridge just as you get to the other end.
\ No newline at end of file
+across the bridge just as you get to the other end.
+
+There's a set of twisting stairs to the left and a long dark tunnel to your right.
+Which way do you go?
+
+Take the [stairs](../stairs/stairs.md).
+Enter the [tunnel](../tunnel/tunnel.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.